README.md

```diff
@@ -1,71 +1,67 @@
-# Getting Started with Create React App
+![Image text](./src/images/nimplayLogo.jpg)
```

Comment:
- [ ] [REQUIRED] Please remove your profile `logo` and paste the Microverse badge!